src/protocol.errorcode.ts

```diff
@@ -16,13 +16,16 @@ export enum ProtocolErrorCode {
     INVALID_TOKEN = 'InvalidToken',
     MISSING_CONFIGURATION = 'MissingConfiguration',
     PLUGIN_ERROR = 'PluginError',
+    FORBIDDEN_EXCEPTION = 'ForbiddenException',
```

Comment:
The error codes are organized by category, so speak.  Is it possible we would alphabetize with in each category?  I believe this would make the code easier to read.

Reply:
Good idea